lib/ash_graphql.ex

```diff
@@ -398,6 +401,13 @@ defmodule AshGraphql do
         attribute_type = unwrap_type(attribute.type)
 
         case attribute_type do
+          Ash.Type.Map ->
+            if attribute.constraints[:fields] do
```

Comment:
You'll probably need to map over the sub-fields the way that the `union` type does.